app/src/main/java/com/duckduckgo/app/tabs/ui/TabGridItemDecorator.kt

```diff
@@ -86,7 +86,7 @@ class TabGridItemDecorator(
     }
 
     companion object {
-        private val BORDER_RADIUS = 9.toPx().toFloat()
+        private val BORDER_RADIUS = 15.toPx().toFloat()
```

Comment:
This is not directly related to the issue, but after working with the decorator I noticed the radius doesn’t match the card view (see the PR description).